query/stdlib/influxdata/influxdb/operators.go

```diff
@@ -14,10 +14,36 @@ import (
 
 const (
 	ReadRangePhysKind     = "ReadRangePhysKind"
+	ReadGroupPhysKind     = "ReadGroupPhysKind"
 	ReadTagKeysPhysKind   = "ReadTagKeysPhysKind"
 	ReadTagValuesPhysKind = "ReadTagValuesPhysKind"
 )
 
+type ReadGroupPhysSpec struct {
+	plan.DefaultCost
+	ReadRangePhysSpec
+
+	GroupMode flux.GroupMode
+	GroupKeys []string
+
+	AggregateMethod string
```

Comment:
I guess this field is not used yet, but will be in the future?

Reply:
Correct.